AUTHORS

```diff
@@ -83,33 +82,33 @@ Gontran Lemaire (gonlem)
 Goodkov Vasiliy Aleksandrovich (goodkov)
 Gregor Cramer
 GuardianRM
-Günther Demetz (pb00067, pb00068)
 Guy Vreuls (gvreuls)
+Günther Demetz (pb00067, pb00068)
```

Comment:
GNU coreutils' `sort` would revert this, because it treats ü as u.

Reply:
Well, I personally think that `ü` should come after `u`. In fact, all letter variations should come after the base letter, such as `ö` after `o`, etc.